schema/v5.0/CVE_JSON_5.0.schema

```diff
@@ -126,15 +130,16 @@
                     "items": {
                         "type": "string",
                         "description": "Name of the affected source code file, function, method, subroutine, or procedure (optional).",
-                        "minLength": 1
+                        "minLength": 1,
+                        "maxLength": 4000
                     }
                 },
                 "collectionURL": {
                     "type": "string",
                     "description": "A URL that, among the users of the software package collection, is considered the most popular starting point for accessing the collection (optional).",
                     "maxLength": 500,
```

Comment:
min/max not required when format is uri

Reply:
removed min/max on URI strings, set "format":"uri"